HELM-7: create an RPM package from the Helm app

RangerRick opened the pull request:
This PR adds a script to generate an RPM package in `dist/packages/`.  If you want to test, it works fine with Homebrew's RPM.

RangerRick commented:
I've updated this to make the package not be 200MB of `node_modules`, and also a few other cleanup tweaks.  ;)

RangerRick commented on makerpm.js:
```diff
@@ -0,0 +1,85 @@
+  fs.mkdirSync(path.join(rpmbuilddir, 'SOURCES'));
+  fs.mkdirSync(path.join(rpmbuilddir, 'RPMS'));
+  fs.mkdirSync(path.join(rpmbuilddir, 'BUILD'));
+  fs.linkSync(path.join('SOURCES', pkginfo.name + '.tar.gz'), path.join(rpmbuilddir, 'SOURCES', pkginfo.name + '.tar.gz'));
```
Ah, yeah, shouldn't be significantly different.  Fixed.

RangerRick commented on makerpm.js:
```diff
@@ -0,0 +1,85 @@
+    {
+      stdio: ['inherit', 'inherit', 'inherit']
+    }
+  );
```
I added a `which` check, and then also exit(1) if the spawn fails.

j-white commented:
My version of node (v6.11.1) doesn't seem to like that `catch` statement you added:

```
$ ./makerpm.js 
/home/jesse/git/grafana-opennms-helm-app/makerpm.js:30
catch (err) {
^^^^^
SyntaxError: Unexpected token catch
```

RangerRick commented:
Haha yeah, turns out you're supposed to push *then* save in your editor... Should be fixed now.  Also fixed permissions to be owned by the `grafana` user.  (Should already be correct in the debian PR.)

j-white commented:
Cool. I had just noticed that permission problem too. Let me give this another whirl.

j-white commented:
I still get tons of these messages when installing:
```
warning: user opennms-helm-app does not exist - using root
warning: group opennms-helm-app does not exist - using root
```
Also, I don't think you meant to include the `opennms-helm-app` directory in this commit.

RangerRick commented:
Yeah, it looks like somewhere deep inside, `specit` is always setting the username to the package name.  I just changed the template to hardcode it to `grafana`.  Also yeah, deleted the helm app directory.  Perils of switching back and forth between the RPM and debian branches.  :)

j-white commented:
Permissions are good now, but I still have a problem with that catch statement:
```
$ ./makerpm.js 
/home/jesse/git/grafana-opennms-helm-app/makerpm.js:30
catch (err) {
^^^^^
SyntaxError: Unexpected token catch
```

RangerRick commented:
Hm, I'm not sure how, since line 30 doesn't look like that anymore; it's got the `}` at the beginning.

j-white commented:
Yep, my bad.
